pytest-selenium/regions/my_highlights.py

```diff
@@ -418,14 +421,14 @@ def print(self):
             button = self.find_element(*self._print_locator)
             Utilities.click_option(self.driver, element=button)
```

Comment:
similar issue, the function is no longer returning a value so the return and doctext should be changed to show that